static/styles/left_sidebar.css

```diff
@@ -343,15 +343,18 @@ li.top_left_recent_topics {
     height: 20px;
 }
 
+/* stylelint-disable */
```

Comment:
Don’t ignore all stylelint rules; ignore individual rules by name instead. And prefer ignoring them for specific lines rather than entire blocks. ([Documentation](https://stylelint.io/user-guide/ignore-code/).)

Reply:
sure, I found out the CSS property i need to disable for allowing `-webkit-box`, 
it's `value-no-vendor-prefix`